core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -161,6 +161,7 @@ private[spark] object UIUtils extends Logging {
 
   def commonHeaderNodes: Seq[Node] = {
     <meta http-equiv="Content-type" content="text/html; charset=utf-8" />
+    <link rel="shortcut icon" href="about:blank"/>
```

Comment:
Definitely I will. Another thing I found that this is not the problem with the standalone spark history server, but also exists in the the live UI running in the driver for a running application.